src/brod_group_coordinator.erl

```diff
@@ -1023,7 +1034,10 @@ log(#state{ groupId  = GroupId
 
 %% @private Make metata to be committed together with offsets.
 -spec make_offset_commit_metadata() -> binary().
-make_offset_commit_metadata() -> coordinator_id().
+make_offset_commit_metadata() ->
+  %% Use a '+1/' prefix as a commit from group member which supports
```

Comment:
why `+1` and not `roundrobin_v2`?

Reply:
mainly to keep it short. it is repeated for each and every offset commit.